src/userProfile/Profile.jsx

```diff
@@ -69,6 +53,14 @@ export const Profile = () => {
     updateUser({ profileUpdated, id, dispatchUser });
   };
 
+  const onSelectAvatar = (e) => {
```

Comment:
i need an explanaition for these one, please, is there a useState for the avatar?

Reply:
Yes, we have one there.